generators/app/index.js

```diff
@@ -88,6 +92,35 @@ module.exports = class extends Generator {
         message: 'Is this a form app?',
         default: false,
       },
+      {
+        type: 'input',
+        name: 'contentRepoLocation',
+        message:
+          'Where can I find the vagov-content repo? This path can be absolute or relative to vets-website. (Leave blank to skip this step.)',
```

Comment:
Maybe instead of `Leave blank to skip this step` something like `Leave blank to skip adding a content page to host your new app`. Or something to make it a little clearer what's being skipped.

Reply:
Ah, yeah. I like that better. I wasn't sure how to make this clear in as few words as possible. :+1: